src/test/groovy/edu/ie3/datamodel/io/source/csv/CsvTestDataMeta.groovy

```diff
@@ -7,22 +7,30 @@ package edu.ie3.datamodel.io.source.csv
 
 import edu.ie3.datamodel.io.naming.FileNamingStrategy
 
+import java.nio.file.Paths
+
 /**
  * Holds meta data for csv tests e.g. file and folder paths
  */
 trait CsvTestDataMeta {
 
-	static String testParticipantsBaseFolderPath = new File(CsvTestDataMeta.getResource('/testGridFiles').toURI()).absolutePath
-	static String testTimeSeriesBaseFolderPath = new File(CsvTestDataMeta.getResource('/testTimeSeriesFiles').toURI()).absolutePath
-	static String graphicsFolderPath = testParticipantsBaseFolderPath.concat(File.separator).concat("graphics")
-	static String typeFolderPath = testParticipantsBaseFolderPath.concat(File.separator).concat("types")
-	static String gridFolderPath = testParticipantsBaseFolderPath.concat(File.separator).concat("grid")
-	static String participantsFolderPath =  testParticipantsBaseFolderPath.concat(File.separator).concat("participants")
-	static String resultEntitiesFolderPath = testParticipantsBaseFolderPath.concat(File.separator).concat("results")
-	static String timeSeriesFolderPath =  testTimeSeriesBaseFolderPath
-	static String thermalFolderPath = testParticipantsBaseFolderPath.concat(File.separator).concat("thermal")
-	static String coordinatesFolderPath = testParticipantsBaseFolderPath.concat(File.separator).concat("coordinates")
+	static String timeSeriesFolderPath = getResourceAbs("timeseries")
+	static String graphicsFolderPath = getResourceAbs("graphics")
+	static String typeFolderPath = getResourceAbs("types")
+	static String participantsFolderPath =  getResourceAbs("participants")
+	static String resultEntitiesFolderPath = getResourceAbs("results")
+	static String thermalFolderPath = getResourceAbs("thermal")
+	static String coordinatesIconFolderPath = getResourceAbs("coordinates_icon")
+	static String coordinatesCosmoFolderPath = getResourceAbs("coordinates_cosmo")
+
+	static String gridDefaultFolderPath = getResourceAbs("grid/grid_default")
```

Comment:
Maybe we should in general have another discussion about the concept of moving the test resources to the equivalent package.

I don't feel fully convinced, yet, as the package structure is additionally complemented with a subordinate directory structure. However, the point, where subordinate directory structure starts and where actual package structure ends, is not fully clear...

Reply:
Well, one could certainly do without the secondary directory structure and just place all resources just at package directory level.
In many cases, this would get confusig really fast though, as with the `io.source.csv` package here, since mance classes are placed here in `src/main`. The additional subdirectories offer a helpful substructure to resources that are needed at that level.

Reply:
That's fine with me

Reply:
I tried your suggestion and improved the structure in some places, let me know what you think